### ISSUE TITLE
Error: Undefined rule rule-empty-line-before

### ISSUE THREAD

**Issue by jonnybarnes** [stylelint/stylelint#2389]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Attempting to run stylelint in the cli gives an error about an undefined rule.

> Which rule, if any, is this issue related to?

`rule-empty-line-before`

> What CSS is needed to reproduce this issue?

All my scss files give the same error.

> What stylelint configuration is needed to reproduce this issue?

I have `stylelint` installed globally, and `stylelint-config-standard` installed locally in the project.

My project’s `.stylelintrc` is as follows:

```json
{
    "extends": "stylelint-config-standard",
    "rules": {
        "indentation": 4
    }
}
```

> Which version of stylelint are you using?

`7.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --syntax=scss path/to/file.scss`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

A report on style violations.

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
stylelint --syntax=scss resources/assets/sass/emoji.scss
Error: Undefined rule rule-empty-line-before
    at module.exports (/usr/local/lib/node_modules/stylelint/lib/utils/configurationError.js:8:27)
    at Object.keys.forEach.ruleName (/usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:276:13)
    at Array.forEach (native)
    at normalizeAllRuleSettings (/usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:272:29)
    at augmentConfigBasic.then.then.then.then.augmentedConfig (/usr/local/lib/node_modules/stylelint/lib/augmentConfig.js:81:12)
```

**Comment by evilebottnawi** [comment]
@jonnybarnes please update to latest version

**Comment by jonnybarnes** [comment]
Worked, `npm update -g` didn’t automatically update to the latest version for some reason.
